azure/datalake/store/transfer.py

```diff
@@ -387,7 +387,8 @@ def _update(self, future):
                     merge_future = self._submit(
                         self._merge, self._adlfs, dst,
                         [chunk for chunk, _ in sorted(cstates.objects,
-                                                      key=lambda obj: obj[1])])
+                                                      key=lambda obj: obj[1])], 
```

Comment:
Use https://docs.python.org/3/library/operator.html#operator.itemgetter to make this cleaner.